factory/core/coreComponents.go

```diff
@@ -227,17 +231,20 @@ func (ccf *coreComponentsFactory) Create() (*coreComponents, error) {
 
 	if genesisNodesConfig.StartTime == 0 {
 		time.Sleep(1000 * time.Millisecond)
+
 		ntpTime := syncer.CurrentTime()
-		genesisNodesConfig.StartTime = (ntpTime.Unix()/60 + 1) * 60
+		startTime := ntpTime.Add(1 * time.Minute)
```

Comment:
I don't think these are equivalent
the old code aligns the start of the round to the next minute boundary
the new code just adds a delta of one minute without alignment.